app/models/conversation.rb

```diff
@@ -8,20 +8,15 @@ class Conversation < ApplicationRecord
     joins(job_application: :vacancy)
       .merge(Vacancy.in_organisation_ids(org_id))
   }
-  scope :with_latest_message_date, lambda {
-    joins(:messages)
-      .select("conversations.*, MAX(messages.created_at) as latest_message_at")
-      .group("conversations.id")
-  }
   scope :with_unread_jobseeker_messages, lambda {
     joins(:messages)
       .where(messages: { type: "JobseekerMessage", read: false })
       .distinct
   }
 
-  def self.default_title_for(job_application)
-    "Regarding application: #{job_application.vacancy.job_title}"
-  end
+  scope :ordered_by_unread_and_latest_message, lambda {
+    order(has_unread_messages: :desc, last_message_at: :desc)
+  }
```

Comment:
As this is built now, it seems like conversations where the Publisher sent the latest message and the Jobseeker has still not read it will also be placed on top of the Publisher conversation list. Since the new `has_unread_messages` doesn't distinguish between `jobseeker` and `publisher`.

Is this a bug or a feature? 😅